pom.xml

```diff
@@ -160,7 +159,6 @@
                 <extensions>true</extensions>
                 <configuration>
                     <compatibleSinceVersion>2.0</compatibleSinceVersion>
-                    <minimumJavaVersion>8</minimumJavaVersion>
```

Comment:
Adapting to the removal of this property from `maven-hpi-plugin` a few releases ago.